torch/distributions/continuous_bernoulli.py

```diff
@@ -13,6 +13,7 @@
     probs_to_logits,
 )
 from torch.nn.functional import binary_cross_entropy_with_logits
+from torch.types import _size
```

Comment:
Did this pass the ruff linter? I would have thought it would have hidden all these behind an if else TYPE_CHECKING?